scene/3d/physics/soft_body_3d.cpp

```diff
@@ -78,8 +81,48 @@ void SoftBodyRenderingServerHandler::close() {
 	write_buffer = nullptr;
 }
 
-void SoftBodyRenderingServerHandler::commit_changes() {
-	RS::get_singleton()->mesh_surface_update_vertex_region(mesh, surface, 0, buffer);
+void SoftBodyRenderingServerHandler::fti_pump() {
+	buffer_prev = buffer;
```

Comment:
You don't need to copy data here, just have two pointers for prev and curr and swap them. Or swap a couple of indices.

Reply:
I didn't notice it was using `Vector`.

Might be worth double checking it's not doing any extra allocation / deallocation as a result, @Ivorforce is the guy to ask here. `unref` seems to call `Memory::free_static`..